tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3877,3 +3871,858 @@ def test_complex128():
+    fl64 = np.float64(randint(-9223372036854775808, 9223372036854775807))
+
+    f_bl = epyccel(test_bool_int, language=language)
+
+    assert (f_bl(True) == test_bool_int(True))
+    assert (f_bl(False) == test_bool_int(False))
+
+    f_integer = epyccel(test_int_int, language=language)
+    f_integer8 = epyccel(test_int8_int, language=language)
+    f_integer16 = epyccel(test_int16_int, language=language)
+    f_integer32 = epyccel(test_int32_int, language=language)
+    f_integer64 = epyccel(test_int64_int, language=language)
+
+    assert (f_integer(integer) == test_int_int(integer))
+    assert (f_integer8(integer8) == test_int8_int(integer8))
+    assert (f_integer16(integer16) == test_int16_int(integer16))
+    assert (f_integer32(integer32) == test_int32_int(integer32))
+    assert (f_integer64(integer64) == test_int64_int(integer64))
+
+    f_fl = epyccel(test_float_int, language=language)
+    f_fl32 = epyccel(test_float32_int, language=language)
+    f_fl64 = epyccel(test_float64_int, language=language)
+
+    assert (f_fl(fl) == test_float_int(fl))
+    assert (f_fl32(fl32) == test_float32_int(fl32))
+    assert (f_fl64(fl64) == test_float64_int(fl64))
+
+def test_numpy_int32(language):
+
+    @types('bool')
+    def test_bool_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('int')
+    def test_int_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('int8')
+    def test_int8_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('int16')
+    def test_int16_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('int32')
+    def test_int32_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('int64')
+    def test_int64_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('float')
+    def test_float_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('float32')
+    def test_float32_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    @types('float64')
+    def test_float64_int32(a):
+        import numpy as np
+        b = np.int32(a)
+        return b
+
+    import numpy as np
+
+    if sys.platform == 'win32':
+        integer = randint(-2147483648, 2147483647)
+    else:
+        integer = randint(-9223372036854775808, 9223372036854775807)
+    integer8 = np.int8(randint(-128, 127))
+    integer16 = np.int16(randint(-32768, 32767))
+    integer32 = np.int32(randint(-2147483648, 2147483647))
+    integer64 = np.int64(randint(-9223372036854775808, 9223372036854775807))
+    fl = np.float(randint(-2147483648, 2147483647))
+    fl32 = np.float32(randint(-2147483648, 2147483647))
+    fl64 = np.float64(randint(-9223372036854775808, 9223372036854775807))
```

Comment:
Why are you using integers to test float? Why not use `rand` instead of `randint`?